documentation/docs.css

```diff
@@ -0,0 +1,34 @@
+@import url('./../styles/style.css');
+main{
+    grid-column: 3 / 13;
+}
+main .component-container{
+    grid-column: 1 / 7;
+    border-bottom: 2px solid var(--blue);
+    margin-bottom: 20px;
+}
+main .component-container .main-heading{
+    color: var(--text-color);
+    font-size: 3rem;
+    font-weight: 300;
+}
+main .component-container .bold-desc{
+    font-family: 'Roboto', sans-serif;
+    font-size: 2rem;
+    font-weight: 200;
+    margin-bottom: 20px;
+}
+main .component-container .small-title{
+    font-size: 2rem;
+    color: var(--text-color);
+}
+main .component-container .component-preview{
+    border: 2px solid var(--white);
+    margin:20px 0;
+    padding: 20px;
+}
+main .component-container .component-code{
+    width: 100%;
+    height: auto;
+    margin: 30px 0;
```

Comment:
Use rem instead of px. It will help with responsiveness.

Reply:
I have changed px to rem. Please approve this.